Automattic/woocommerce-payments
# refactor: settings hooks

Fixes #

#### Changes proposed in this Pull Request









Talked about in p1621459594014300-slack-C01RTUVR81K and https://github.com/Automattic/woocommerce-payments/pull/1824#discussion_r637867957 .

- Using `[value, setValue]` format for the existing hooks
- Using array (instead of objects with properties) like the `useState` React hooks allows for better minification

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Go to your WCPay Home: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- No functionality has changed

-------------------
